### PR TITLE
Updated 2.6 release doc

### PR DESCRIPTION
**Pull request by owaiskazi19**
### Description
Removed changes related to extensibility from the 2.6 release doc

### Issues Resolved
[List any issues this PR will resolve]

### Check List
- [ ] New functionality includes testing.
  - [ ] All tests pass
- [ ] New functionality has been documented.
  - [ ] New functionality has javadoc added
- [ ] Commits are signed per the DCO using --signoff
- [ ] Commit changes are listed out in CHANGELOG.md file (See: [Changelog](../blob/main/CONTRIBUTING.md#changelog))

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: release-notes/opensearch.release-notes-2.6.0.md [hunk 1]
```diff
@@ -5,13 +5,10 @@
 - Add index create block when all nodes have breached high disk watermark ([#5852](https://github.com/opensearch-project/OpenSearch/pull/5852))
 - Add cluster manager throttling stats in nodes/stats API ([#5790](https://github.com/opensearch-project/OpenSearch/pull/5790))
 - Add support for feature flags in opensearch.yml ([#4959](https://github.com/opensearch-project/OpenSearch/pull/4959))
-- Add query for initialized extensions ([#5658](https://github.com/opensearch-project/OpenSearch/pull/5658))
```
Comment (kotwanikunal) [inline release-notes/opensearch.release-notes-2.6.0.md:8]: These were also removed from the CHANGELOG from the 2.x section: https://github.com/opensearch-project/OpenSearch/blob/main/CHANGELOG.md
Can you please add it back if it's unreleased?

Comment (owaiskazi19) [reply release-notes/opensearch.release-notes-2.6.0.md:8]: Thanks for noticing it. Pushed the change!

File: CHANGELOG.md [hunk 2]
```diff
@@ -68,6 +68,9 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 ### Security
 
 ## [Unreleased 2.x]
+- Add query for initialized extensions ([#5658](https://github.com/opensearch-project/OpenSearch/pull/5658))
```
Comment (andrross) [inline CHANGELOG.md:71]: These changes are on the 2.6 branch, right? Why are they going back into the changelog as unreleased?

Comment (owaiskazi19) [reply CHANGELOG.md:71]: We are planning to release experimental version of extensibility in 2.8/2.9. If you think we should keep these changes in 2.6 release note, I'm fine with that.

Comment (owaiskazi19) [reply CHANGELOG.md:71]: Let me know if this PR needs to be closed

Comment (andrross) [reply CHANGELOG.md:71]: Well, this change is being released in 2.6 right? If this is worthy of being detailed in the release notes it should stay in. If this is just one incremental piece of a larger feature to be announced in a future release then maybe we can just remove it?

Comment (owaiskazi19) [reply CHANGELOG.md:71]: This is an incremental change of a larger feature for the future release. Removing it sounds good to me for 2.6.

Comment (andrross) [reply CHANGELOG.md:71]: Sounds good to me. Worth keeping in mind for all the incremental PRs that are yet to come as well :)

**Comment by owaiskazi19** [comment]
We are planning to release extensibility with incremental features. Closing this PR.
cc: @saratvemulapalli

**Pull request closed without merging**